### ISSUE TITLE
[Dev Asks} Crystals need name!

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#5461]
So I am adding crystals as a higher tier resources to the game. No worry I also plan to add various other resources type.
I need some cool names for them.

For now we will have crystal for:
- Lightcruise and gravity units
- Warp drive

Please share any name you fancy! A good name would ideally:
- give a hint on the their use
- be easy to pronounce
- convey the idea it is a mineral / crystal. So finish with yst, ite, yl and so on.

Cheers

**Comment by tsunamayo** [comment]
From Uncle-Ulty:

I believe the crystal should receive a unique name that makes it special. I'd avoid a generic name or a known name. It must be a name that would be related to the Starship Evo itself.

Why am I saying this? because the lightdrive travel is prbly the most important and most recurrent feature that players gonna use. Something as important and special such as a lightdrive cruise, that's only possible with a lightdrive engine, must receive an important name.

-e.g.

-evonium, evoryx, evotite, evocite, EVOLUX, evolium, ... you got the idea...

**Comment by tsunamayo** [comment]
From Janssendel:

As for the crystals, we need something that's easy to remember and unique, phonetically speaking. Crystals of a lower tier should sound more "normal" compared to the bizarre names of the highest tiered crystals. You can only use a letter once if they are at the front of the word to avoid confusion.

Low Quality: Elkite, Trinitite, Algolite, Omberlux
Standard Quality: Netenkite, Snyral, Rholite, Hromium
High Quality: Leprochite, Dulcinium, Corralux, Uvinium
Exceptional Quality: Morphite, Fordinium, Pneumatite, Gildinium, Zitenkite, Quoralux, Btenki

Why 4 tiers? Crystals of Exceptional quality are rare to come by but can be fed to reactors to gain a buff to certain stats:
-Morphite allows some regeneration of the ship's hull and an increase in ship hull points.

-Gildinium
allows the shields of the ship to better scatter attacks, also making the shield yellow/golden.

-Btenki allows the propulsion systems of the ship to use less fuel and warp further.

-Zitenkite gives all weapons a significant buff in damage (with added shake when projectiles hit)

(E) Grade crystals can be used as a substitute to regular fuel that can be used for a long time before running out, but their scarcity makes them a resource viable only during the most desperate fights.

**Comment by tsunamayo** [comment]
from Coaster-block:

@tsunamayo Do you want more scifi names or something "regular"? I was thinking simple "low tier" crystals could be references to real-life materials. Such as Quartz or variants of Aluminium crystals. (Emeralds, Rubies, Saphires.)

My suggestion, based on real elements and things from Physics.
Tier 1: Quartz, Alumite, Carbonite. (Silicon, Aluminium, Carbon)

Tier 2: Hydronium, Yttricite, Kor. (Hydrogen Metal, Yttrium, Pottasium.)

Tier 3: Xenium, Neutronite, Promethyx. (Xenon, Neutrons, Prometheum.)

Tier 4: Gravitorium, Resonon, Zero, Queo, (Gravity, Resonance and Zero-point energy, Quantum Entanglement)

Tier 5: ZXN, (can only be crafted by using all other minerals. Cannot be found. It would be the game's "Unobtanium/miracle material." )

**Comment by Alkaliii** [comment]
Yumerria
Yumerrium
Kofum Crystals
Dokus Crystals
Zumarock
Likalyte
Lithokir
Juna Jems
Monohydrium
Bilithorhesium
Noxdivum Chloride
Vanverium
Zamzitye

**Comment by SovietSponge** [comment]
Evonite

**Comment by Huttrem** [comment]
well guess huttrenyte is not good idea, what about Evohuttnyte or Evutnyte, what resource will use shield's? protonite cool be cool name

**Comment by ProPeach** [comment]
Maybe something derived from Latin that means fast/travel/door/quick/energy kinda thing

So Zephyrite, 
Veloxite/Velozite
Indonite (from "bridge" for warping? Like a bridge between systems...)
Stellanite
Navitite

**Comment by Tackietacktack** [comment]
Many things get names from Explorer. I haven't enough knowledge to know where discovered anti Gravity or lightspeed. Fizeau, Einstein, hawking, Podkletnov?
Fizeauium, fizeau crystal
Einsteinnum
Hawkingite
Podkletnovium, podkletnizit

**Comment by BigBadKangaroo** [comment]
Hi there,

For lightcruise:
-Luxyl
-Celerite
(I also really like previous mentions: "Evonite","Zephyrite" and " Stellanite")

For gravity units:
-Gravonite

For warp:
-Solarite
-Warp Shards (something different, but it could work aswell)

**Comment by tsunamayo** [comment]
> Maybe something derived from Latin that means fast/travel/door/quick/energy kinda thing
> 
> So Zephyrite, Veloxite/Velozite Indonite (from "bridge" for warping? Like a bridge between systems...) Stellanite Navitite

Yes it was also what I wanted to do for warp, something with astra.

**Comment by tsunamayo** [comment]
> Many things get names from Explorer. I haven't enough knowledge to know where discovered anti Gravity or lightspeed. Fizeau, Einstein, hawking, Podkletnov? Fizeauium, fizeau crystal Einsteinnum Hawkingite Podkletnovium, podkletnizit

Well this game happens in a galaxy far far away...

**Comment by tsunamayo** [comment]
> Hi there,
> 
> For lightcruise: -Luxyl -Celerite (I also really like previous mentions: "Evonite","Zephyrite" and " Stellanite")
> 
> For gravity units: -Gravonite
> 
> For warp: -Solarite -Warp Shards (something different, but it could work aswell)
> 
> *I would prefere names that are easy to remember and not too complicated

Yes I really like those, I am itching toward celerite. Warp shards also works really well!

**Comment by Kaiser-Indrasil** [comment]
**Newtonite** - for gravity units and lightcruise fuel - to commemorate the real-life discoverer of gravity. Also a provisional non-systematic name of an undiscovered element with the symbol Nw and atomic number 119.

**Foldium** - for warp drives - simple play of the word "fold", synonym for "warp".
